### DISCUSSION TITLE
Możliwość pobierania parametrów w  /sale/offers

### DISCUSSION THREAD

**Discussion by bugproof** [allegro/allegro-api#4390] [Suggestion - verification]
:bulb: **Sugestia**

Fajnie gdyby można było jeszcze wybrać sobie parametry jakie mają zostać dołączone w `/sale/offers` np. numery katalogowe części 215858. Np `includeParameters=215852,inneid` albo `includeParameters=true`

Aktualnie gdy chcę pobrać numery katalogowe części wszystkich ofert muszę robić 2x więcej zapytań (`/sale/offers` + `/sale/offers/{offerId}`). W aktualnym rozwiązaniu mam osobne cache które jest aktualizowane raz na jakiś czas aby zmniejszyć ilość wysyłanych zapytań, jednak fajnie gdyby można było wyciągać rzeczywiste dane na dany czas bezpośrednio z listy ofert bez robienia dodatkowych zapytań.

**Comment by PrzemyslawLukanowski** [comment]
Dziękuję za sugestię. Zgodnie z [naszym newsem](https://github.com/allegro/allegro-api/issues/4337) przeniosłem wątek do zakładki [Discussions](https://github.com/allegro/allegro-api/discussions).
